onnxruntime/hosting/environment.cc

```diff
@@ -10,33 +10,31 @@
 namespace onnxruntime {
 namespace hosting {
 
-HostingEnvironment::HostingEnvironment(onnxruntime::logging::Severity severity) : logger_id_("HostingApp"),
-                                                                                  default_logging_manager_(
-                                                                                      std::unique_ptr<onnxruntime::logging::ISink>{&sink_},
-                                                                                      severity,
-                                                                                      /* default_filter_user_data */ false,
-                                                                                      onnxruntime::logging::LoggingManager::InstanceType::Default,
-                                                                                      &logger_id_) {
-  auto status = onnxruntime::Environment::Create(this->runtime_environment_);
+HostingEnvironment::HostingEnvironment(logging::Severity severity) : severity_(severity),
+                                                                     default_filter_user_data_(false),
+                                                                     logger_id_("HostingApp"),
+                                                                     default_logging_manager_(
+                                                                         std::unique_ptr<logging::ISink>{new LogSink{}},
+                                                                         severity,
+                                                                         false,
```

Comment:
When we call a function with boolean, it will be easier for the reader if we could add a comment to mention what is the original parameter name.

Reply:
```suggestion
                                             /* default_filter_user_data */ false
```